revenue_timeseries.yaml

```diff
@@ -171,30 +153,29 @@ SELECT
       AND CURRENT ROW
   ) AS cumulative_cogs
 FROM
-  __daily_revenue
+  daily_revenue
 WHERE
   date BETWEEN '2023-12-01'
   AND '2023-12-31'
 ORDER BY
   date DESC;
 "
-  # For eval sample nlimtiaco_sc_6__0
+ # For eval sample nlimtiaco_sc_6__0
   - name: "lowest revenue each month"
-    question: For each month, what was the lowest daily revenue and on what date did
-      that lowest revenue occur?
+    question: "For each month, what was the lowest daily revenue and on what date did that lowest revenue occur?"
     sql: "WITH monthly_min_revenue AS (
         SELECT
           DATE_TRUNC('MONTH', date) AS month,
           MIN(daily_revenue) AS min_revenue
-        FROM __daily_revenue
+        FROM daily_revenue
```

Comment:
@sfc-gh-sgolebiewski can we confirm that we are postprocess vqrs with appending `__` now?

Reply:
Hmm, how can I confirm that? 🤔